entity-services/src/main/xdmp/entity-services/entity-services.xqy

```diff
@@ -149,29 +152,29 @@ declare function es:extraction-template-generate(
  : Generates an element for configuring Search API applications
  : Intended as a starting point for developing a search grammar tailored
  : for entity and relationship searches.
- : @param An entity type document.
+ : @param An entity model
  :)
 declare function es:search-options-generate(
-    $model
+    $model as map:map
```

Comment:
Can you please let me know Charles, the difference here... $model Vs $model as map:map.

Looks like earlier you were verifying all the models using ensure-model() to be map:map. Now that its removed and validation is separate, are we assuming here that $model is a valid map:map representation?
Is it because model-validate() and model-create() check and return the error which ensure-model() used to earlier?